web/pages/components/Navbar.tsx

```diff
@@ -0,0 +1,199 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Menu,
+  MenuButton,
+  MenuList,
+  Image,
+  Stack,
+  Text,
+  Link,
+  AccordionButton,
+  AccordionItem,
+  Accordion,
+  AccordionPanel,
+  Divider,
+} from "@chakra-ui/react";
+import { ChevronDownIcon, HamburgerIcon } from "@chakra-ui/icons";
+import { useAuth } from "../../context/auth";
+import { trpc } from "../../utils/trpc";
+export default function Navbar() {
+  const { user } = useAuth();
+  const userRole = trpc.user.getRole.useQuery({ uid: user?.uid ?? null });
+  if (user !== null) {
```

Comment:
Flip this to return `<></>` when `user === null`. Have the default return be the navbar JSX.